pkg/skaffold/initializer/init.go

```diff
@@ -44,6 +45,14 @@ func DoInit(ctx context.Context, out io.Writer, c config.Config) error {
 		return err
 	}
 
+	// helm projects can't currently be bootstrapped automatically by skaffold, so we fail fast and link to our docs instead.
+	if len(a.ChartPaths()) > 0 {
+		//nolint
+		return errors.New(`Projects set up to deploy with helm must be configured with skaffold manually.
```

Comment:
A bit shorter, and they ran `skaffold init`, so configuring Skaffold is implicit.
```suggestion
		return errors.New(`Projects set up to deploy with helm must be manually configured.
```